apps/api/views.py

```diff
@@ -245,6 +245,22 @@ def _get_id(data):
         ScheduledMessage.objects.bulk_create(new)
 
 
+update_state_serializer = inline_serializer(
+    name="update_state_serializer",
+    fields={
+        "state": serializers.JSONField(),
+    },
+)
+
+update_state_response_serializer = inline_serializer(
+    name="update_state_response",
+    fields={
+        "success": serializers.BooleanField(),
+        "new_state": serializers.JSONField(),
```

Comment:
nit: I think this should just be `state`
```suggestion
        "state": serializers.JSONField(),
```